### ISSUE TITLE
Feedback on library groups for packages libraries

### ISSUE THREAD

**Issue by bwilkerson** [dart-lang/language#1808]
The [Packaged Libraries](https://github.com/dart-lang/language/blob/master/working/packaged-libraries/feature-specification.md) proposal introduces the notion of _library groups_ and includes the following:

> __TODO: This feature seems like a user chore without much user value. We could simply say that macros cannot be in library cycles. Is this worth keeping? Are there other things library groups would be useful for (like inferring Blaze targets or defining ABI boundaries)?__

I agree with the assessment that the feature requires a lot of effort from users for little or no value. I believe that having implicit library groups computed by the SCC computation would have equal value for no effort. The computation is already being done by all of the tools, and the results of the computation could be used for anything that an explicit library group could be used for.

**Comment by munificent** [comment]
Yeah, I'm leaning in this direction too (in large part because of you initial comment to this effect back on the old modules doc). I'll leave it as-is for now so we can discuss it more, but I think the most likely option is to remove it from the proposal.
